resources/js/components/facilities/GridList.vue

```diff
@@ -0,0 +1,32 @@
+<script setup lang="ts">
+import DataGrid from '../DataGrid.vue'
+
+const facilityData = [
```

Comment:
Your data should also be strongly typed. Use the same logic we used to build type and interface in "GridList" component.
You should write your type/interface then use the type like this
```
const facilityData: facilityDataInterface = [
    ...
]
```